.gitignore

```diff
@@ -1,3 +1,5 @@
 *.pdf
 node_modules
-.env
\ No newline at end of file
+.env
+package.json
```

Comment:
Why ignore package.json?

Reply:
This is because the environment for my commitizen need to install a package in the root folder of the project. (I don't know why but if I don't do this it won't work.)

The `package.json` I ignore is `/ntu-book/package.json` instead of `/ntu-book/src/backend/package.json`  so I suppose it to be fine